tests/acceptance/usage-in-acceptance-test.js

```diff
@@ -0,0 +1,33 @@
+import { test } from 'qunit';
+import moduleForAcceptance from '../../tests/helpers/module-for-acceptance';
+import { visit, click, fillIn, waitUntil } from 'ember-native-dom-helpers/test-support/helpers';
+
+moduleForAcceptance('Acceptance | usage in acceptance');
+
+test('Usage awaiting the world to settle', async function(assert) {
```

Comment:
Let me amend that thing above. 
In order for async/await to work you have to have the regenerator runtime (via `ember-maybe-import-regenerator`), include the babel polyfill or use the new targets feature (https://github.com/ember-cli/rfcs/blob/master/complete/0095-standardise-targets.md) in ember-cli 2.13 and only support browsers with async/await.